bindata/oauth-openshift/authentication-clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:openshift-authentication
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin  # TODO fix, this is madness
```

Comment:
could you please change the comment to: oauth-server needs to read secrets in all namespaces so there is no point trying to tighten its permissions

Reply:
done